pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -8870,6 +8878,9 @@ spec:
 			if err != nil {
 				t.Fatalf("Got an error getting reconciled run out of fake client: %s", err)
 			}
+			tt.expectedPipelineRun.Status.Provenance = &v1beta1.Provenance{
+				FeatureFlags: alphaFeatureFlags,
+			}
```

Comment:
is there a reason for doing this instead of adding it directly in the expectedpipelinerun yaml?

adding it to the yaml ensures that there's only one place to check for the expected pipelinerun status

Reply:
Thanks for the feedback. Done.

No particular for unit tests, but we do need to use variables in e2e tests instead of adding it in yamls.